lambda_functions/codecommit/MyCodeCommitFunction.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+# snippet-sourcedescription:[MyCodeCommitFunction.py demonstrates how to use an AWS Lambda function to return the URLs used for cloning an AWS CodeCommit repository to a CloudWatch log.]
+# snippet-service:[codecommit]
+# snippet-keyword:[Python]
+# snippet-keyword:[AWS CodeCommit]
+# snippet-keyword:[Code Sample]
+# snippet-keyword:[GetRepository]
+# snippet-sourcetype:[full-example]
+# snippet-sourcedate:[2016-03-07]
+# snippet-sourceauthor:[AWS]
+# snippet-start:[codecommit.python.MyCodeCommitFunction.py]
```

Comment:
You do not need to have `.py` at the end, but you can add a `.complete`

Reply:
Changed as requested.

Reply:
Thank you